schema/components.json

```diff
@@ -18,14 +17,18 @@
         "type": {
           "$ref": "urn:devicetype-library:generated-schema#/definitions/console-port/properties/type"
         },
-        "poe": {
-          "type": "boolean"
+        "description": {
+          "type": "string"
+        },
+        "_is_power_source": {
```

Comment:
is the property _is_power_source used in netbox? as this can only be changed if that is the property netbox is expecting.

Reply:
@m-hau Apologies just realised that this is for validation and not something that is actually imported into netbox